src/folderlistmodel.cpp

```diff
@@ -364,6 +432,23 @@ FolderListModel::FolderStandardType FolderListModel::folderTypeFromId(const QMai
     return NormalFolder;
 }
 
+bool FolderListModel::isStandardFolder(const QMailFolderId &id) const
+{
+    FolderStandardType folderType = folderTypeFromId(id);
+    return folderType == InboxFolder || folderType == DraftsFolder
```

Comment:
JunkFolder is not in this list - intentional?
Maybe "return folderType != NormalFolder" would be better?